engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-manage-ostree-branches.html

```diff
@@ -0,0 +1,52 @@
+<span page-title ng-model="repository">{{ 'Manage Ostree Branchesfor Repository:' | translate }} {{ repository.name }}</span>
+
+<a ui-sref="products.details.repositories.info({productId: product.id, repositoryId: repository.id})">
+  <i class="fa fa-angle-double-left"></i>
+  {{ "Back to Repository Details" | translate }}
+</a>
+
+<div data-extend-template="layouts/details-nutupane.html">
+
+  <div data-block="pane-loading"></div>
+
+  <div data-block="messages">
+    <div bst-alerts success-messages="successMessages" error-messages="errorMessages"></div>
+
+    <div bst-alert="success" ng-hide="generationTaskId === undefined">
+      <button type="button" class="close" ng-click="clearTaskId()">&times;</button>
+      <p translate>
+        Ostree Branch metadata generation has been initiated in the background.  Click
+        <a ng-href="{{ taskUrl() }}">Here</a> to monitor the progress.
+      </p>
+    </div>
+  </div>
+
+  <div data-block="header">
+    <h3 translate>Ostree Branches in {{ repository.name }}</h3>
+  </div>
+
+  <div data-block="table">
+    <table class="table table-striped table-bordered" >
+
+      <thead>
+        <tr bst-table-head row-select>
```

Comment:
Can I do anything with these branches other than browse them? If not, row-select is not needed.